bqn-glyphs.el

```diff
@@ -71,3 +71,4 @@
 	  (bqn-glyph-mode))
         (set-window-buffer window buffer)
         (fit-window-to-buffer window)))))
+(provide 'bqn-glyphs)
```

Comment:
New files should be added in a single commit. Squash those smaller mistakes.

Reply:
Being more accurate: if you are adding a new file, it should not be split between commits.